src/node_i18n.cc

```diff
@@ -788,7 +788,8 @@ static int GetColumnWidth(UChar32 codepoint,
       if (ambiguous_as_full_width) {
         return 2;
       }
-      // Fall through if ambiguous_as_full_width if false.
+      // If ambiguous_as_full_width is false:
+      // Fall through
```

Comment:
Unrelated change?

Reply:
No, it's the second commit.  `// Fall through` is a magic comment that gcc recognizes.

Reply:
ah.... I don't think I ever knew that... good to know :-)